client/hwlib/src/collectors/os_info.rs

```diff
@@ -84,9 +84,20 @@ impl KernelPackage {
     }
 }
 
-pub(crate) fn get_architecture(runner: &impl CommandRunner) -> Result<String> {
-    let arch = runner.run_command(DPKG, &["--print-architecture"])?;
-    Ok(arch.trim().to_owned())
+pub(crate) fn get_deb_arch(arch: &str) -> Result<String> {
+    let deb_arch = match arch {
+        "aarch64" => "arm64",
+        "arm" => "armel",
+        "loongarch64" => "loong64",
+        "mips" => "mips",
+        "powerpc" => "powerpc",
+        "powerpc64" => "ppc64",
+        "riscv64" => "riscv64",
+        "x86" => "i386",
+        "x86_64" => "amd64",
+        _ => arch,
+    };
```

Comment:
instead of supporting this mapping, let's just do the way checkbox does it and use `dpkg` command